batch/batch/front_end/front_end.py

```diff
@@ -810,7 +812,11 @@ async def create_batch(request, userdata):
     async def insert(tx):
         rows = tx.execute_and_fetchall(
             '''
-SELECT * FROM billing_project_users
+SELECT *, (
```

Comment:
Is the reason for doing the subquery like this so you don't have to do a group by?

Reply:
I did it like this because I need to lock the billing_projects table to check that there's not the exclusive lock from the `_close_billing_project` transaction; not sure if there's a better way to do it.

Reply:
@danking Can you comment on this query?

Reply:
Thanks for the explanation! I'll rewrite it as an inner join.